app/Repositories/RecensioniRepository.php

```
@@ -54,7 +55,7 @@ public function canUserReview(int $userId, int $libroId): bool
             return $row['count'] > 0;
 
         } catch (\Throwable $e) {
-            error_log("Error checking if user can review: " . $e->getMessage());
+            SecureLogger::error('[RecensioniRepository] canUserReview failed', ['exception' => $e->getMessage()]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'canUserReview'. (undefined)

(StaticAccess)